README.md

```diff
@@ -164,4 +190,5 @@ explain why.
 
 ## A2.7
 
-«replace this with your answer»
+Type 2. It is context-less as no word is dependent on the word before or after it,
+yet is not quite normal as the adjectives and adverbs can be empty. 
```

Comment:
Because we can parse it with a state machine, it is type 3. The next token _always_ determines what we do next, so it is deterministic.

# -3